microsoft/PowerToys
# ImageResizer new icon

## Summary of the Pull Request

Use the new icon:

![image](https://user-images.githubusercontent.com/3206696/92448501-f7175780-f1b8-11ea-8471-cb1fd20ffd10.png)


## PR Checklist
* [x] Applies to #https://github.com/microsoft/PowerToys/issues/3903

## Validation Steps Performed

Build solution, build msi, install msi, verify new cion is used (may require a Windows reboot to reset the icon cache)
